<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditParamIntegrationMasterClass</class>
 <widget class="QMainWindow" name="EditParamIntegrationMasterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditParamIntegrationMaster</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpdateDefaultButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>580</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Overwrite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>580</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>580</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFileName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>611</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="General">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="QFrame" name="frameGlobalParam">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>611</width>
        <height>431</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Slave">
     <attribute name="title">
      <string>Slave</string>
     </attribute>
     <widget class="QListWidget" name="listWidgetSlaveList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>421</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSlaveListPlus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>51</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Slave</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSlaveListMinus">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>460</y>
        <width>51</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>461</width>
        <height>491</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Slave machine code</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxMachineCode">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Slave IP address (host name)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditSlaveAddress">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>201</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Shared folder</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditSharedFolder">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>291</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonShooseSharedFolder">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>100</y>
         <width>24</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonImagePathPlus">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>450</y>
         <width>51</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonImagePathMinus">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>450</y>
         <width>51</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Minimum additional second</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxMinAdditionalSec">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>130</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <number>99999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>154</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Maximum additional second</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxMaxAdditionalSec">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>154</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <number>99999999</number>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidgetImageFolders">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>441</width>
         <height>211</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>ImagePath</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Offset for sync</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxOffsetXForSync">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>190</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <number>99999999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxOffsetYForSync">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>190</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <number>99999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>190</y>
         <width>16</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>190</y>
         <width>16</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxDependent">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Dependent</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxSlaveIsInMaster">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Slave is In Master</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
